scheme/common/utils.go

```diff
@@ -35,3 +38,24 @@ func GetMandatoryPathSegment(key string, fields map[string]*structpb.Value) (str
 
 	return segment, nil
 }
+
+func GetPubKeyFromPEMEncodedPubKey(ta []byte) (crypto.PublicKey, error) {
```

Comment:
since this is shared code, we should add some suitable godoc comment.

also please change `ta` to something more meaningful meaningful